packages/react-composites/tests/browser/meeting/MeetingComposite.test.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { dataUiId } from '../common/utils';
+import { test } from './fixture';
+import { expect, Page } from '@playwright/test';
+
+export const waitForMeetingCompositeToLoad = async (page: Page): Promise<void> => {
+  await page.waitForLoadState('load');
+
+  // @TODO
+  // We wait 3 sec here to work around flakiness due to timing.
+  // It sometimes take a while for the local video / audio streams to load in CI environments.
+  // We don't have a good way to know when the composite is fully loaded.
+  await page.waitForTimeout(3000);
```

Comment:
Can the fix you added to call composites be applied here as well?
That waits for video state